Content.Server/_Goobstation/Heretic/Components/EldritchItemComponent.cs

```diff
@@ -0,0 +1,9 @@
+using Content.Server.Heretic.EntitySystems;
+
+namespace Content.Server.Heretic.Components;
+
+[RegisterComponent, Access(typeof(EldritchItemSystem))]
+public sealed partial class EldritchItemComponent : Component
+{
+    [DataField] public bool Spent = false;
```

Comment:
false by default - `= false` is redundant, also put `[DataField]` on its own line pleasies